src/ContentComponents/Svg/index.tsx

```diff
@@ -12,20 +12,27 @@ export type SvgComponentData = {
 export const SvgComponent: FC<ComponentArgs<SvgComponentData>> = ({ data, onChange }) => {
   const { editing, api } = useWhppt();
 
+  const [created, setCreated] = useState(false);
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState('');
   const [svgString, setSvgString] = useState('');
 
+  const loadSvgString = useCallback(() => {
+    api.gallery
+      .loadSvg(data?.svg?.galleryItemId)
+      .then(setSvgString)
+      .catch(err => setError(err.message || err))
+      .finally(() => setLoading(false));
+  }, [api.gallery, data?.svg?.galleryItemId]);
+
   useEffect(() => {
+    if (!created) return setCreated(true);
+
     if (data?.svg?.galleryItemId) {
       setLoading(true);
```

Comment:
setLoading can move the the loadSvgString function. That way if we have to call it from anywehere else the loading state still gets set.

Reply:
done